R/axis-secondary.R

```diff
@@ -160,7 +160,7 @@ AxisSecondary <- ggproto("AxisSecondary", NULL,
   # Inherit settings from the primary axis/scale
   init = function(self, scale) {
     if (self$empty()) return()
-    if (!is.function(self$trans)) abort("transformation for secondary axes must be a function")
+    if (!is.function(self$trans)) cli::cli_abort("transformation for secondary axes must be a function")
```

Comment:
```suggestion
    if (!is.function(self$trans)) cli::cli_abort("Transformation for secondary axes must be a function")
```

Could we make this refer to the original argument name?

Reply:
we could... but it is a function call into an argument which makes it a bit clumsy.. e.g. you would usually do `scale_x_continuous(..., sac.axis = sec_axis(trans = <x>, ...))`

Reply:
since `trans` is also an argument to the scale function you'd have to qualify the naming and I feel the prose version above is less clumsy than `The trans argument in sec_axis() passed to scale_x_continuous() must be a function` or something like that